.travis.yml

```diff
@@ -1,6 +1,19 @@
 language: rust
 rust:
     - nightly
+
+before_script:
+    - "sh -e /etc/init.d/xvfb start"
+    - "export DISPLAY=:99.0"
+    - "wget http://selenium-release.storage.googleapis.com/2.50/selenium-server-standalone-2.50.1.jar"
+    - "java -jar selenium-server-standalone-2.50.1.jar > /dev/null &"
+    - sleep 5
+    - curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.30.2/install.sh | bash
+    - nvm install 4.2
+    - nvm use 4.2
+    - npm install selenium-webdriver
+
 script:
     - cargo build
     - RUST_BACKTRACE=1 cargo test
+    - ((cargo run ; true) > /dev/null & node test/selenium/ftu_test.js && killall -HUP foxbox) || (killall -HUP foxbox ; exit 1)
```

Comment:
My bash is not that strong, I imagine this line can be improved to look nicer. I did test that it fails/passes on Travis if the test fails/passes, and that in both cases the cargo server is killed, and its exit code is ignored.